Resources/Locale/en-US/forensics/forensics.ftl

```diff
@@ -0,0 +1,15 @@
+forensic-scanner-interface-title = Forensic scanner
+forensic-scanner-interface-fingerprints = Fingerprints
+forensic-scanner-interface-fibers = Fibers
+forensic-scanner-interface-no-data = No scan data available
+forensic-scanner-interface-print = Print
+forensic-scanner-report-title = Forensics Report
+forensic-pad-unused = It hasn't been used.
+forensic-pad-sample = It has a sample: {$sample}
+forensic-pad-gloves = Ask {CAPITALIZE($target)} to take off {POSS-ADJ($target)} gloves.
```

Comment:
Probably could say "target is wearing gloves" as you may not want to bother asking them in certain situations.

Reply:
```suggestion
forensic-pad-gloves = {CAPITALIZE($target)} is wearing gloves.
```